apps/builder/app/builder/features/seo/project-settings.tsx

```diff
@@ -132,24 +128,72 @@ const ProjectSettingsContent = (props: Props) => {
           rows={5}
           autoGrow
           maxRows={10}
-          value={props.value.code ?? ""}
+          value={props.meta.code ?? ""}
           onChange={handleChange("code")}
         />
       </Grid>
-      <div />
-    </Grid>
+    </>
+  );
+};
+
+const ProjectAdvancedSettings = (props: {
+  settings: ProjectSettings;
+  onSettingsChange: (settings: ProjectSettings) => void;
+}) => {
+  const ids = useIds(["atomicStyles"]);
+
+  const handleChange =
+    <T extends keyof ProjectSettings>(name: T) =>
+    (val: ProjectSettings[T]) => {
+      props.onSettingsChange({
+        ...props.settings,
+        [name]: val,
+      });
+    };
+
+  return (
+    <>
+      <Separator />
+      <Grid gap={2} css={{ mx: theme.spacing[5], px: theme.spacing[5] }}>
+        <Label sectionTitle>Advanced Settings</Label>
```

Comment:
```suggestion
        <Label sectionTitle>Publish Settings</Label>
```

Reply:
this way the section is not everything, but specifically for publishing settings

Reply:
more specific it could be Build Settings or Compiler Settings